app/components/feedback/container_component.html.erb

```diff
@@ -1,16 +1,18 @@
 <div class="p-2 border shadow-lg rounded-lg max-w-prose mx-auto flex-1">
-  <div class="flex justify-between items-center">
-    <%= heading(class: "text-lg") %>
-    <%= render Feedback::EditButtonComponent.new(feedback:, current_user: user) %>
-  </div>
-  <div>
-    <% if feedback.completed? %>
-      <div class="p-2">
-        <p><span class="font-bold">Rating:</span> <%= feedback.overall_rating %>/10</p>
-      </div>
-      <%= render Feedback::QuestionComponent.with_collection(questions) %>
-    <% else %>
-      <p class="p-4">Awaiting completion of feedback.</p>
-    <% end %>
-  </div>
+  <%= tag.turbo_frame(id: dom_id(feedback)) do %>
+    <div class="flex justify-between items-center">
+      <%= heading(class: "text-lg") %>
+      <%= render Feedback::EditButtonComponent.new(feedback:, current_user: user) %>
```

Comment:
I know this is merged, but out of curiousity, what is the difference between using the `user` var passed to the container component and `current_user`? Would it be more descriptive to rename `user` to `current_user`?